molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/ElasticsearchRepository.java

```
@@ -38,8 +38,7 @@ public EntityMetaData getEntityMetaData()
 	@Override
 	public void rebuildIndex()
 	{
-		throw new UnsupportedOperationException(); // FIXME reindex from source
```

Why do you silently want to ignore this exception?